my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
```

Comment:
it is a style choice but: import matplotlib as mpl helps to know where the functions in the program come from, and it would correspond to import numpy as np